### PR TITLE
[MMDS][CDAP-13383] Fix directives representation for a model consistent

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Shows a hint of directives applied to a model and provides a "View" link for the user to view it in a tabular view.

JIRA: https://issues.cask.co/browse/CDAP-13383

### CODE REVIEW

File: cdap-ui/app/cdap/components/Experiments/DetailedView/ModelsTable/index.js [hunk 1]
```diff
@@ -235,10 +238,10 @@ const renderModelDetails = (model, newlyTrainingModel, experimentId) => {
           <div>{model.description || '--'}</div>
         </div>
         <div>
-          <strong># Directives </strong>
+          <strong># Directives ({directivesCount}) </strong>
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/Experiments/DetailedView/ModelsTable/index.js]: from the mock it seems like we don't need the '#' anymore?

**Comment by tonybach** [comment]
One minor question, besides that code LGTM :+1:

File: cdap-ui/app/cdap/components/Experiments/DetailedView/ModelsTable/index.js (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/Experiments/DetailedView/ModelsTable/index.js]: 👍 Removed `#`.

**Comment by ajainarayanan** [comment]
Demoed to @bdmogal . Merging.

**Pull request merged**